Extensions/DialogueTree/dialoguetools.js

```diff
@@ -266,7 +271,7 @@ gdjs.dialogueTree.getLineOptionsTextVertical = function(optionSelectionCursor) {
  * @returns {number} The number of options
  */
 gdjs.dialogueTree.getLineOptionsCount = function() {
-  if (this.options.length) {
+  if (!this.dialogueIsRunning || this.options.length) {
```

Comment:
Are you sure about your condition?

Reply:
yes, this prevents it from trying to read data when no dialogue is initiated

Reply:
oops, that is actually an error, will amend now